cdc/processor/pipeline/sink.go

```diff
@@ -269,12 +268,11 @@ func (n *sinkNode) Receive(ctx pipeline.NodeContext) error {
 		}
 	case pipeline.MessageTypeCommand:
 		if msg.Command.Tp == pipeline.CommandTypeStopAtTs {
-			if msg.Command.StoppedTs < n.checkpointTs {
-				log.Warn("the stopped ts is less than the checkpoint ts, "+
-					"the table pipeline can't be stopped accurately, will be stopped soon",
-					zap.Uint64("stoppedTs", msg.Command.StoppedTs), zap.Uint64("checkpointTs", n.checkpointTs))
-			}
 			n.targetTs = msg.Command.StoppedTs
+			n.barrierTs = msg.Command.StoppedTs
+			if err := n.flushSink(ctx, msg.Command.StoppedTs); err != nil {
```

Comment:
`n.flushSink(ctx, n.resolvedTs)` is called in other places, could this still flush more events beyond `global resolved ts`

Reply:
I believe that by design no event is ever flushed beyond `global resolved ts`.